docs/api-reference/aggregation-layers/contour-layer.md

```diff
@@ -134,13 +134,16 @@ A very small z offset that is added for each vertex of a contour (Isoline or Iso
 
 * Default: `object => object.position`
 
-Method called to retrieve the position of each point.
+Method called to retrieve the position of each object.
```

Comment:
Nit: I think of data as arrays of `rows` but I guess we already use the terminology `object` so probably best to stick with that...